oss-internship-2020/curl/custom_curl/custom_curl.cc

```diff
@@ -0,0 +1,33 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <iostream>
+#include "custom_curl.h"
+#include <stdlib.h>
+
+CURLcode curl_easy_setopt_ptr(CURL* handle, CURLoption option, void* parameter) {
+    return curl_easy_setopt(handle, option, parameter);
```

Comment:
Please reformat this file using clang-format and the "Google" style.

In short:
* Two spaces for indent
* Hard 80-character limit
* Include order: see above

Reply:
For more details, see teh Google C++ style guide: https://google.github.io/styleguide/cppguide.html